roles/monitoring_client/tasks/install_node_exporter.yml

```diff
@@ -10,12 +18,22 @@
 - name: Download and unarchive node_exporter
   ansible.builtin.unarchive:
     src: "{{ monitoring_client_node_exporter_binary }}"
-    dest: "{{ monitoring_client_node_exporter_install_dir }}"
+    dest: "{{ monitoring_client_node_exporter_download_dir }}"
     remote_src: true
     owner: "{{ monitoring_client_owner }}"
     group: "{{ monitoring_client_group }}"
     extra_opts: "--strip-components=1"
-    creates: "{{ monitoring_client_node_exporter_install_dir }}/node_exporter"
+  tags:
+    - molecule-idempotence-notest
```

Comment:
The idempotence test always fails at this task - I think because the firewall is started and configured (the error is `Request failed: <urlopen error [Errno -2] Name or service not known>`).